docs/iris/src/whatsnew/3.0.4.rst

```diff
@@ -0,0 +1,597 @@
+.. include:: ../common_links.inc
+
+v3.0.4 (?? ??August 2021)
+*************************
+
+This document explains the changes made to Iris for this release
+(:doc:`View all changes <index>`.)
+
+
+.. dropdown:: :opticon:`alert` v3.0.4 Patches
+   :container: + shadow
+   :title: text-primary text-center font-weight-bold
+   :body: bg-light
+   :animate: fade-in
+   :open:
+
+   The patches included in this release include:
+
+   🐛 **Bugs Fixed**
+
+   #. `@pp-mo`_ reverted a change made previously in (:pull:`3659`) to
+      :meth:`iris.fileformats.pp.PPDataProxy.__getitem__`. The check for empty slicings
+      is no longer needed since (:pull:`4135`) was added. (:pull:`4141`)
```

Comment:
@pp-mo Why have you moved this `whatsnew` entry?

It was part of the 3.0.2 release. We can't change patch release history.

I was expecting a whatnew entry relating to this PR #4233 or #4124.

Reply:
Sorry I seem to have messed this up somehow with in a merge/rebase process.
Also the original proposed new entry ("bugs in cube printout") has completely vanished.
Will address.

Reply:
I must say, having a continously-augmented "lastest.rst" would avoid these problems.
Why can we not use that within bugfix branches as we do in the main branch ??